platform-python.spec

```diff
@@ -1470,57 +1106,101 @@ fi
 %{pylibdir}/xml
 
 %if "%{_lib}" == "lib64"
-%attr(0755,root,root) %dir %{_prefix}/lib/python%{pybasever}
-%attr(0755,root,root) %dir %{_prefix}/lib/python%{pybasever}/site-packages
-%attr(0755,root,root) %dir %{_prefix}/lib/python%{pybasever}/site-packages/__pycache__/
+%attr(0755,root,root) %dir %{_prefix}/lib/%{name}%{pybasever}
+%attr(0755,root,root) %dir %{_prefix}/lib/%{name}%{pybasever}/site-packages
+%attr(0755,root,root) %dir %{_prefix}/lib/%{name}%{pybasever}/site-packages/__pycache__/
 %endif
 
 # "Makefile" and the config-32/64.h file are needed by
 # distutils/sysconfig.py:_init_posix(), so we include them in the core
 # package, along with their parent directories (bug 531901):
 %dir %{pylibdir}/config-%{LDVERSION_optimized}-%{_arch}-linux%{_gnu}/
 %{pylibdir}/config-%{LDVERSION_optimized}-%{_arch}-linux%{_gnu}/Makefile
-%dir %{_includedir}/python%{LDVERSION_optimized}/
-%{_includedir}/python%{LDVERSION_optimized}/%{_pyconfig_h}
+%dir %{_includedir}/platform-python%{LDVERSION_optimized}/
+%{_includedir}/platform-python%{LDVERSION_optimized}/%{_pyconfig_h}
 
 %{_libdir}/%{py_INSTSONAME_optimized}
-%{_libdir}/libpython3.so
-%if 0%{?with_systemtap}
-%dir %(dirname %{tapsetdir})
-%dir %{tapsetdir}
-%{tapsetdir}/%{libpython_stp_optimized}
-%doc systemtap-example.stp pyfuntop.stp
-%endif
+%{_libdir}/libplatform-python3.so
+
+%files libs-devel
+%defattr(-,root,root,-)
```

Comment:
It's in all the %files sections in python3. I agree we should remove it, but I'd say first in python3.